tests/product/test_collect.py

```diff
@@ -294,3 +274,33 @@ def test_collect_logs_server_not_installed(self):
                            self.PA_ONLY_CLUSTER)
         self.upload_topology()
         self._assert_no_logs_downloaded()
+
+    def test_collect_logs_multiple_server_logs(self):
+        self.setup_cluster(NoHadoopBareImageProvider(),
+                           self.STANDALONE_PRESTO_CLUSTER)
+        self.run_prestoadmin('server start')
+        self.cluster.write_content_to_host('#!/bin/bash\n touch '
+                                           '/var/log/presto/server.log-2',
```

Comment:
Please put touch on the next line so it's with its argument
Also, do we need to touch another log file to ensure that we fetch multiple logs if server.log doesn't exist? If not, please comment explaining why.